data/css/all-aboard.css

```diff
@@ -97,6 +97,28 @@ a:focus {
                 opacity 1s;
 }
 
+.user-data {
+    text-align: left;
+}
+
+.user-data p {
+    overflow: hidden;
+    line-height: 1;
+    padding: 10px 10px 10px 10px;
+    margin: 0 0;
+}
+
+.user-data p:hover {
+    background-color: #e8e8e8;
+    border: .5px solid gray;
```

Comment:
I reckon we can drop sub pixel sizing and just go with 1 as this will not be noticeable on non highDPI displays. Thoughts?